client/src/InfoBase/index.hbs.html

```diff
@@ -43,12 +43,12 @@
             </div>
           {{/unless}}
           <div class="row">
-            <div class="col-md-4 col-xs-8">
+            <div class="col-md-4 col-8">
               <a href="{{logo_link}}" class="ib-site-header-logo gl-FIP" aria-label="{{logo_a11y_label}}">
                 <object type="image/svg+xml" tabindex="-1" role="img" data="{{CDN_URL}}/svg/sig-blk-{{lang}}.svg" aria-label="{{logo_a11y_label_2}}" class="canada-logo"></object>
               </a>
             </div>
-            <div class="col-md-8 col-xs-4">
+            <div class="col-md-8 col-4">
```

Comment:
Hmm, we could avoid having to re-publish these files if we re-add support for `col-xs-`, at least 4 and 8 (maybe with some big comments explaining that they're for temporary legacy index support, and shouldn't be used elsewhere).

Is removing the divs below necessary, or just cleanup? If it was necessary, could be get around it with some other specific CSS elsewhere?

Reply:
If we're keeping them around, the right place to stick them would be `wet-bootstrap3-holdovers.scss`.